peps/pep-0772.rst

```diff
@@ -325,32 +320,28 @@ for two consecutive Council elections automatically removes a person from the li
 Electors, until they re-submit their intention to resume their participation to the Packaging Council in
 writing.
 
-In exceptional circumstances, it may be necessary to remove someone from the Electors against their will (for
-example: egregious and ongoing code of conduct violations). An Elector may be removed by a two-thirds majority
-vote by the Packaging Council (in practice: 4:1 for a Council with five members).
-
-If the relevant Elector is also on the Packaging Council, then they can participate in the vote. They are
-removed from the Packaging Council if the vote removes them as an Elector. The vacancy is filled as per the
-`normal process <vacancy>`_.
-
 Vote of no confidence
 ---------------------
 
-In exceptional circumstances, the Electors may remove a sitting Council member, or the entire Council, via a
-vote of no confidence.
+In exception circumstances, a vote of no confidence may be called to remove a sitting Packaging
+Council member, or the entire council.  The Python Steering Council may call such votes of no
+confidence, with no second being necessary.  Anyone may request such a vote of no confidence from
+the Steering Council, regardless of the requester's membership or affiliation, and the Steering
+Council has full discretion to call for the vote or not.
 
-A no-confidence vote is triggered when an Elector calls for one publicly on an appropriate public
-communication channel, and another Elector seconds the call within one week.
-
-The vote lasts for two weeks. Each Elector votes for or against. If at least two thirds of Electors express a
-lack of confidence, then the vote succeeds. Quorum for a vote of no confidence is 50% of Electors.
+The vote of no confidence lasts for two weeks. Each Elector votes for or against. If at least two
+thirds of Electors express a lack of confidence, then the vote succeeds. Quorum for a vote of no
+confidence is 50% of Electors.
 
 There are two forms of no-confidence votes: those targeting a single member, and those targeting the Council
 as a whole. The initial call for a no-confidence vote must specify which type is intended. If a single-member
 vote succeeds, then that member is removed from the Council and the resulting vacancy can be handled by the
 `normal process <vacancy>`_. If a whole-Council vote succeeds, the Council is dissolved and a new Council
 election is triggered immediately, using the rules for `whole Council <whole-council>`_ elections.
 
+If a no-confidence vote fails, another identical no-confidence vote (that is, of the same form, and
+against the same member for the first form) may not be raised for six months from the initial call.
+
```

Comment:
Sorry to go back and forth on this, but now that it's the Steering Council triggering these, we don't really need a rate limit. (But we can keep it too, it's fine.)

Reply:
Good call.  Agreed it's unnecessary, so I'm going to strike it.